report/firefox_code_coverage/codecoverage.py

```diff
@@ -69,27 +70,28 @@ def _save_tasks(response):
 
 def download_binary(url, path, retries=5):
     """Download a binary file from an url"""
-    for i in range(1, retries + 1):
+    @tenacity.retry(stop=tenacity.stop_after_attempt(retries),
```

Comment:
Set this annotation directly on `download_binary`, so you don't have to define an inner function.

Reply:
But then how do we input the retries number to the decorator? I have written an inner function in order to pass the retries to the decorator. I am sorry if this is a silly question, this is my first time using decorators.

Reply:
It's not a silly question at all! We can simplify the function and just remove the argument and hardcode the number of retries.

Reply:
sure i have made the changes